docs/website/docs/guides/deployment-configurations/bare-metal.md

```diff
@@ -59,7 +59,7 @@ See [generate.sh](https://github.com/openxla/iree/blob/main/runtime/src/iree/hal
 for example command-line instructions of some common architectures
 
 You can replace the MLIR file with the other MLIR model files, following the
```

Comment:
L52 - L62: end with `.` for each sentence?

Reply:
Added to the sentences, omitted for the bulleted list items. Looks like general guidance is to add periods for list items, but I've been omitting them. I'll keep the consistency for now.

https://developers.google.com/style/periods#bulleted-lists
https://developers.google.com/style/lists#capitalization-and-end-punctuation